src/server/resolve.rs

```diff
@@ -133,3 +137,90 @@ impl StoreAddrResolver for MockStoreAddrResolver {
         unimplemented!();
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
We can mock `PdClient` for tests, and we already have a `TestPdClient` in `tests/raftstore/pd.rs`.